Design/readme.md

```diff
@@ -1,15 +1,48 @@
 # CBS Mockup
-The MVP mockup is made using [Balsamiq mockups](https://balsamiq.cloud/sgyjn/ppbia): 
+The [MVP Mockup](https://balsamiq.cloud/sgyjn/ppbia)
 
-You can also find the source file and a clickable PDF in this folder, but please note that the PDF contains a couple of blank pages. Refer to the Balsamiq mockup above to view these. 
+The [1.0 Mockup](https://balsamiq.cloud/sgyjn/pz5f9qp) 
```

Comment:
Is this the MVP Mockup? If so, could we call it the MVP Mockup so that we stay consistant through out the project when talking about the MVP?

Reply:
I will pull from upstream and do a better readme tonight to make it more clear